lib/data_fetchers/git_fetcher.rb

```diff
@@ -38,18 +38,31 @@ def line_changes
   end
 
   def languages
-    languages = github.languages.map do |language, percent|
+    languages = github.languages
+      .merge(gitlab.languages) { |key, value1, value2| ((value1 + value2) / 2.0).round }
```

Comment:
Yepp, using the absolute numbers gives proper result, mathematically.

But we still need to take care not to calculate mirrored projects twice.